api/src/main/java/org/killbill/billing/subscription/api/SubscriptionBaseInternalApi.java

```diff
@@ -80,4 +80,7 @@ public SubscriptionBaseBundle createBundleForAccount(UUID accountId, String bund
 
     public void updateExternalKey(UUID bundleId, String newExternalKey, InternalCallContext context);
 
+    public Iterable<DateTime> getFutureNotificationsForAccount(InternalCallContext context);
+
+    public Map<UUID, DateTime> getNextFutureEventForSubscriptions(final SubscriptionBaseTransitionType eventType, final InternalCallContext internalCallContext);
 }
```

Comment:
Confused why those appear in that PR? I added those APIS recently (see https://github.com/killbill/killbill/commit/15519aa1c28e3360dac7e46753c48fa2155329c7). Would be nice to ONLY see the changes coming from you guys to keep it readable.